ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -371,7 +370,9 @@ private class JdbcLedgerDao(
                     recordTime = recordTime,
                     completionInfo = state
                       .CompletionInfo(actAs, applicationId, commandId, None, submissionId),
-                    reasonTemplate = reason.toParticipantStateRejectionReason,
+                    reasonTemplate = reason.toParticipantStateRejectionReason(errorFactories)(
+                      new DamlContextualizedErrorLogger(logger, loggingContext, None)
```

Comment:
Why do we not pass the `submissionId` as the third parameter here?

Reply:
87c9b918072712fed1b6224060a2b39f34cf6feb